airtime_mvc/application/models/Show.php

```diff
@@ -1432,39 +1429,70 @@ public static function getMaxLengths()
     }
 
     public static function getStartEndCurrentMonthView() {
-        $first_day_of_calendar_month_view = mktime(0, 0, 0, date("n"), 1);
-        $weekStart = Application_Model_Preference::GetWeekStartDay();
-        while (date('w', $first_day_of_calendar_month_view) != $weekStart) {
-            $first_day_of_calendar_month_view -= 60*60*24;
-        }
-        $last_day_of_calendar_view = $first_day_of_calendar_month_view + 3600*24*42;
 
-        $start = new DateTime("@".$first_day_of_calendar_month_view);
-        $end = new DateTime("@".$last_day_of_calendar_view);
```

Comment:
These getStartEndCurrentBlahView() functions were all unused before, as scheduleEventPreloadAction() was being called, but the data was never being used.

I rewrote them because they don't properly handle the user timezone.